packages/strapi-utils/lib/content-types.js

```diff
@@ -108,6 +108,63 @@ const isMediaAttribute = attr => {
   return (attr.collection || attr.model) === 'file' && attr.plugin === 'upload';
 };
 
+const getKind = obj => obj.kind || 'collectionType';
+
+const pickSchema = model => {
+  const schema = _.cloneDeep(
+    _.pick(model, ['connection', 'collectionName', 'info', 'options', 'attributes'])
+  );
+
+  schema.kind = getKind(model);
+  return schema;
+};
+
+const formatContentType = (
+  model,
+  { modelName, modelOrigin, defaultConnection },
+  { apiName, pluginName } = {}
+) => {
+  const contentType = {
+    __schema__: pickSchema(model),
+    kind: getKind(model),
+    modelType: 'contentType',
+    modelName,
+    globalId: model.globalId || _.upperFirst(_.camelCase(modelName)),
+    connection: model.connection || defaultConnection,
+    privateAttributes: getPrivateAttributes(model),
+  };
+
+  switch (modelOrigin) {
+    case 'api':
+      Object.assign(contentType, {
+        uid: `application::${apiName}.${modelName}`,
+        apiName,
+        collectionName: model.collectionName || modelName.toLocaleLowerCase(),
+      });
+      break;
+    case 'admin':
+      Object.assign(contentType, {
+        uid: `strapi::${modelName}`,
+        plugin: 'admin',
+        identity: model.identity || _.upperFirst(modelName),
```

Comment:
wondering if this is still used somewhere 🤔  I know it is for controllers but for models we might be able to remove it. can you check that ?

Reply:
You're right, not useful anymmore